src/lib/index/wb_cache.cpp

```diff
@@ -241,6 +246,25 @@ folly::Future< bool > IndexWBCache::async_cp_flush(IndexCPContext* cp_ctx) {
 void IndexWBCache::do_flush_one_buf(IndexCPContext* cp_ctx, IndexBufferPtr buf, bool part_of_batch) {
     LOGTRACEMOD(wbcache, "cp {} buf {}", cp_ctx->id(), buf->to_string());
     buf->set_state(index_buf_state_t::FLUSHING);
+
+#ifdef _PRERELEASE
+
+    if (cp_ctx->is_abrupt()) {
+        LOGTRACEMOD(wbcache, "The cp {} is abrupt! for {}", cp_ctx->id(), BtreeNode::to_string_buf(buf->raw_buffer()));
+        return;
+    }
+    if (auto it = crashing_buffers.find(buf);it != crashing_buffers.end()) {
+        const auto& reasons = it->second;
+        std::string formatted_reasons = fmt::format("[{}]", fmt::join(reasons, ", "));
+        LOGTRACEMOD(wbcache, "Buffer {} is in crashing_buffers with reason(s): {} - Buffer info: {}",
+                    buf->to_string(), formatted_reasons, BtreeNode::to_string_buf(buf->raw_buffer()));
```

Comment:
We don't need another fmt::format over other format right? You could merge them.